client/accept-invite/invite-message/index.jsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import React from 'react'
+
+/**
+ * Internal dependencies
+ */
+import SimpleNotice from 'notices/simple-notice'
+import { dismissInviteAccepted, dismissInviteDeclined } from 'accept-invite/actions'
+import store from 'accept-invite/invite-message/store'
+
+export default React.createClass( {
+
+	getInitialState: function() {
+		return store.get();
+	},
+
+	componentWillMount() {
+		store.on( 'change', () => this.setState( this.getInitialState ) );
+	},
+
+	componentWillUnmount() {
+		store.off( 'change', () => this.setState( this.getInitialState ) );
+	},
+
+	render() {
+		if ( ! this.props.sites ) {
+			return null;
+		}
+		const { accepted, declined, siteId } = this.state;
+		if ( accepted ) {
+			const site = this.props.sites.getSite( siteId );
+			if ( ! site ) {
+				return null;
+			}
+			return (
+				<SimpleNotice status="is-success" onClick={ dismissInviteAccepted }>
+					<h3 className="invite-message__title">{ this.translate( 'You\'re now a user of: %(site)s', { args: { site: site.slug } } ) }</h3>
```

Comment:
I think the site should be a link to the front-end. Also the sentence would read better without the `:` — `You're now a user of <a>Blabla</a>.`, with a period at the end.